CHANGELOG.md

```diff
@@ -6,6 +6,10 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+## [5.3.3] - 2020-08-18
+### Changed
+- Release process is updated to ensure that the published Ruby Gem matches a tag in this repository, so that consumers of this gem can always reference the correct source code included in any given version. [cyberark/conjur-api-ruby](https://github.com/cyberark/conjur-api-ruby/issues/173)
```

Comment:
please add line breaks
```suggestion
- Release process is updated to ensure that the published Ruby Gem matches a tag in this repository,
  so that consumers of this gem can always reference the correct source code included in any given version.
  [cyberark/conjur-api-ruby#173](https://github.com/cyberark/conjur-api-ruby/issues/173)
```